services/books.js

```diff
@@ -3,6 +3,7 @@
 var memoize = require('memoizeasync'),
     db = require('./db'),
     logger = require('./logger'),
+    Console = require('console'),
```

Comment:
No need to require console. You can always call `console.log("something")` in javascript.